diff --git a/business_responsive_theme.theme b/business_responsive_theme.theme
index d4f3208..e7edd80 100644
--- a/business_responsive_theme.theme
+++ b/business_responsive_theme.theme
@@ -120,7 +120,7 @@ function business_responsive_theme_preprocess_page(&$variables) {
   //To get the current URL
   $current_url  = \Drupal::request()->getRequestUri();
   $current_path = explode("/", $current_url);
-  $variables['current_path'] = $current_path[2];
+  $variables['current_path'] = isset($current_path[2]) ? $current_path[2] : current($current_path);
   
   
   // Slider
